x-pack/legacy/plugins/siem/public/containers/case/api.ts

```diff
@@ -238,7 +238,7 @@ export const pushToService = async (
     `/api/action/${connectorId}/_execute`,
     {
       method: 'POST',
-      body: JSON.stringify({ params: casePushParams }),
+      body: JSON.stringify({ params: { action: 'pushToService', actionParams: casePushParams } }),
```

Comment:
What do you think about creating a specific API request per actions. Maybe something like that, I am sure we can find better naming but you get the idea.

- `/api/action/${connectorId}/_push_to_service`
- `/api/action/${connectorId}/_health_check`
- `/api/action/${connectorId}/_update_case_from_service`
- `/api/action/${connectorId}/_is_case_out_dated_from_service`